### ISSUE TITLE
Sort feed items by date

### ISSUE THREAD

**Issue by AaronAcerboni** [AaronAcerboni/feed-mixer#1]
Feed items appear in non-chronological order.

**Comment by bw4518** [comment]
Yes, it would help, but I suppose you are not developing this anymore (?).

**Comment by AaronAcerboni** [comment]
Not in active development. I might look into it at some point but I
wouldn't hold my breath.
